lib/index.d.ts

```diff
@@ -0,0 +1,124 @@
+import type { Boom } from "@hapi/boom";
+import type { AssertionError } from "assert";
+
+export type BounceErrorType = Error | "system" | "boom" | Record<any, any>;
```

Comment:
`Error` is incorrect, it should be `ErrorConstructor`. This doesn't actually change anything, since `Record<any, any>` matches both `Error` and `ErrorConstructor`.